code/src/components/TaskList.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { Task } from 'components/Task'
+
+export const TaskList = () => {
+    const tasks = useSelector((state) => state.ToDoList.tasks)
+    console.log(tasks)
+
+    return (
+        <div>
+        <ul>
+            {tasks.map((task) => ( <Task key={task.id} task={task} /> 
+            ))}
```

Comment:
This should either be all on one line:

```js
{tasks.map((task) => <Task key={task.id} task={task} />)}
```

Body of the map function should be on a new line:

```js
{tasks.map((task) => (
  <Task key={task.id} task={task} /> 
))}
```

It's kind-of a mix of the two currently..